File: openmp_part_I/omp_loop.f90
program omp_loop1

   use omp_lib
   integer :: my_thread
!$omp parallel default(none) private(my_thread)
   my_thread = omp_get_thread_num()
!$omp do
   do i=0,15
      print *,"Thread ",my_thread," gets i = ",i
   end do
!$omp end do nowait

!$omp end parallel

end program omp_loop1

File: openmp-wg-pII-2015/omp_loop.c
//
//

#include <stdio.h>
#include <omp.h>

int main() 
{
    #pragma omp parallel default(none)
    {
        int i;
        int my_thread = omp_get_thread_num();
        #pragma omp for
        for (i=0; i<16; i++)
            printf("Thread %d gets i = %d\n",my_thread,i);
    }
}
